be/src/service/service_be/lake_service.h

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 #pragma once
+#include <span>
```

Comment:
Note: When you cherry-pick this patch to branch-3.1, since branch-3.1 still uses C++17, the use of std::span here will fail to compile, and you will need to change to using arrays instead of std::span in branch-3.1.